pkg/api/v1/conversion.go

```diff
@@ -276,6 +276,7 @@ func Convert_v1_PodStatusResult_To_api_PodStatusResult(in *PodStatusResult, out
 		if err := json.Unmarshal([]byte(value), &values); err != nil {
 			return err
 		}
+		// TODO: Is it OK to modify in in Convert function?
```

Comment:
The way we decode, it is safe.  Decode either creates a new object (v1) or is decoding into, in which case mutating the in object is undefined.